src/floki_selector_lexer.xrl

```diff
@@ -22,6 +22,7 @@ Rules.
 \({ODD}\)                            : {token, {pseudo_class_odd, TokenLine}}.
 \({EVEN}\)                           : {token, {pseudo_class_even, TokenLine}}.
 \({PSEUDO_PATT}\)                    : {token, {pseudo_class_pattern, TokenLine, remove_wrapper(TokenChars)}}.
+\({QUOTED}\)                         : {token, {pseudo_class_quoted, TokenLine, remove_wrapper(remove_wrapper(TokenChars))}}.
```

Comment:
You are right guys. It works good. 👍 

@buhman I think there is no way to use groups in leex today. What would make a little bit fancier is to write another function that removes double wrappers. But I think it's fine this way.